### PR TITLE
Feature/gramine

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
-version=7.1.0
-iexecCommonVersion=6.0.0
-iexecBlockchainAdapterVersion=7.1.1
-iexecResultVersion=7.1.0
-iexecSmsVersion=7.1.0
+version=gramine
+iexecCommonVersion=gramine-NEXT-SNAPSHOT
+iexecBlockchainAdapterVersion=gramine-NEXT-SNAPSHOT
+iexecResultVersion=gramine-NEXT-SNAPSHOT
+iexecSmsVersion=gramine-NEXT-SNAPSHOT
```
Comment (jbern0rd) [inline gradle.properties:5]: do not forget to revert this :pray:

**Review by jbern0rd — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (jeremyjams) [inline gradle.properties:5]:
```suggestion
version=7.1.0
iexecCommonVersion=6.0.1
iexecBlockchainAdapterVersion=7.1.1
iexecResultVersion=7.1.0
iexecSmsVersion=7.1.0
```

File: gradle.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 version=7.1.0
-iexecCommonVersion=6.0.0
+iexecCommonVersion=6.0.1
 iexecBlockchainAdapterVersion=7.1.1
 iexecResultVersion=7.1.0
 iexecSmsVersion=7.1.0
```
Comment (jbern0rd) [inline gradle.properties:5]: Shouldn't we use `-NEXT-SNAPSHOT` dependencies ?

Comment (jbern0rd) [inline gradle.properties:5]: Shouldn't some of those dependencies be in `-NEXT-SNAPSHOT` ?
I see at least `iexecCommonVersion` and `iexecSmsVersion`.

Comment (jeremyjams) [reply gradle.properties:5]: I agree :)

Comment (jeremyjams) [inline gradle.properties:5]:
```suggestion
iexecCommonVersion=6.0.1-NEXT-SNAPSHOT
iexecBlockchainAdapterVersion=7.1.1
iexecResultVersion=7.1.0
iexecSmsVersion=7.1.0-NEXT-SNAPSHOT
```

**Pull request merged**